rust/operator-binary/src/airflow_controller.rs

```diff
@@ -965,7 +915,25 @@ fn build_server_rolegroup_statefulset(
         };
         airflow_container.readiness_probe(probe.clone());
         airflow_container.liveness_probe(probe);
-        airflow_container.add_container_port("http", resolved_port.into());
+        airflow_container.add_container_port(HTTP_PORT_NAME, http_port.into());
+    }
+
+    if let Some(listener_class) =
+        airflow.merged_listener_class(airflow_role, &rolegroup_ref.role_group)
+    {
+        // all listeners will use ephemeral volumes as they can/should
+        // be removed when the pods are *terminated* (ephemeral PVCs will
+        // survive re-starts)
```

Comment:
We should use persistent volumes for web UIs IMO, since upstream load balancers will depend on probably end up hard-coding the target addresses.

Reply:
7ac6077